src/models/guide.cpp

```diff
@@ -1,4 +1,4 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+﻿/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
What is this change?

Reply:
> $file guide.cpp
> guide.cpp: c program text, Unicode text, UTF-8 (with BOM) text

I think it's adding BOM, which shouldn't happen. Which text editor are you using? VSCode shows clearly if a file is _with BOM_ or not

Reply:
I believe @flodolo is right, though not sure how it happened. I use Qt Creator and thought I might've had some messed up auto-format settings but everything looks good.

Anyways, I will revert this!